Source/PeterKottas.DotNetCore.WindowsService/ServiceRunner.cs

```diff
@@ -203,7 +215,7 @@ private static void Install(HostConfiguration<SERVICE> config, ServiceController
                     GetServiceCommand(config.ExtraArguments),
                     cred,
                     autoStart: true,
-                    startImmediately: true,
+                    startImmediately: config.StartImmediately ?? true,
```

Comment:
if it's nullable it should be config.StartImmediately.HasValue ? config.StartImmediately.Value : true instead. But I think it's better to make it non nullable like mentioned in my previous comment.